widgets/PythonEvaluator-transform.json

```diff
@@ -14,6 +14,33 @@
           "widget-attributes": {
             "default": "def transform(record, emitter, context):\n    \"\"\"Transforms the provided input record into zero or more output records (as dictionaries) or errors (as an error dictionary).\n\n    Input records are available as a dictionary.\n\n    Args:\n        record (dict): The input record as a dictionary. e.g. to access a field called 'total' from the input record, use record['total'].\n        emitter (Emitter): an object that can be used to emit zero or more records (using the emitter.emit() method) or errors (using the emitter.emitError() method).\n        context (Context): an object that provides access to:\n                           1. CDAP Metrics - context.getMetrics().count('output', 1);\n                           2. CDAP Logs - context.getLogger().debug('Received a record');\n                           3. Lookups - context.getLookup('blacklist').lookup(input.id); or\n                           4. Runtime Arguments - context.getArguments.get('priceThreshold')\n\n    \"\"\"\n\n    emitter.emit(record)\n"
           }
+        },
+        {
+          "widget-type": "select",
+          "label": "Execution Mode",
+          "name": "executionMode",
+          "widget-attributes": {
+            "values": [
+              "Interpreted",
+              "Native"
+            ],
+            "default": "Interpreted"
+          }
+        }
+      ]
+    },
+    {
+      "label": "Native Mode Settings",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Python Binary",
+          "name": "pythonBinary"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "PYTHONPATH",
+          "name": "pythonPath"
```

Comment:
Let's add a default value here of 'python'.

Even though there is a default in the code, it is still nice from a user perspective to have the default filled in the UI. Our newer plugins follow this pattern where backend default values are also specified in the UI widget.